### DISCUSSION TITLE
Option to display / save failed probes

### DISCUSSION THREAD

**Discussion by uBadRequest** [projectdiscovery/httpx#357] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
I'm looking to keep track of which subdomains fail or timeout or don't have any ports open.

**Describe the solution you'd like**
Option to log all failed requests or failed to resolve domain

**Comment by ehsandeep** [comment, marked as answer]
@uBadRequest 

```
echo hackerone.com | httpx -json | jq .failed
```

```
  "failed": false
```

If you want CLI output, you can use `-probe` flag.
